testing/sdk_metrics.py

```diff
@@ -0,0 +1,36 @@
+'''Utilities relating to checking services' metrics'''
+
+import shakedown
+
+import sdk_hosts as hosts
+import sdk_utils as utils
+import json
+import ast
+
+def get_metrics(service_name, task_name):
+    """Return a list of metrics datapoints.
+
+    Keyword arguments:
+    service_name -- the name of the service to get metrics for
+    task_name -- the name of the task whose agent to run metrics commands from
+    """
+    host = hosts.system_host(service_name, task_name)
+    auth_token = shakedown.run_dcos_command('config show core.dcos_acs_token')[0].strip()
+
+    service_containers_cmd = """curl --header "Authorization: token={}"
+        -s http://localhost:61001/system/v1/metrics/v0/containers""".format(auth_token).replace("\n", "")
```

Comment:
I'm surprised there's no shakedown or dcos commandline utility to help with fetching these urls.